src/NuGet.Core/NuGet.Packaging/Signing/Verification/NuGetSignatureAllowListObject.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging.Signing
+{
+    public abstract class NuGetSignatureAllowListObject
```

Comment:
Why prefix this type name and those below with NuGet?  We're already under a NuGet namespace.